orangecontrib/imageanalytics/http2_client.py

```diff
@@ -25,8 +26,11 @@ class Http2Client(object):
     _no_conn_err = "No connection with server, call reconnect_to_server()"
 
     def __init__(self, server_url, server_port):
-        self._server_url = server_url
-        self._server_port = server_port
+        self._server_url = getenv('ORANGE_EMBEDDING_API_URL', server_url)
+        self._server_port = getenv('ORANGE_EMBEDDING_API_PORT', server_port)
```

Comment:
URL means that the value usually also includes a port (eg. example.com:443). Is a separate variable for port really needed? Either rename the variable to ..API_SERVER or, even better, make everything a bit simpler and get rid of separate ..._API_PORT and include port into the ...API_URL.

I prefer the second option, it is also a default way for constructing a HTTP20 connection (look at http://hyper.readthedocs.io/en/latest/api.html#http-2, port is marked as optional and the default value is 443, which will be the case when we switch to server implementation in k8s).